src/pages/EarnTri/Manage.tsx

```diff
@@ -340,7 +346,7 @@ export default function Manage({
             </ButtonPrimary>
 
             <ButtonPrimary
-              disabled={earnedAmount == null || earnedAmount?.equalTo(ZERO)}
+              disabled={earnedAmount == null || (earnedAmount?.equalTo(ZERO) && doubleRewardAmount?.equalTo(ZERO))}
```

Comment:
If `earnedAmount` is null and `doubleRewardAmount` is `greaterThan(ZERO)`, will this still be disabled (eg for a nonTri pool) ?

Reply:
If you've tested claiming then LGTM, accepting to unblock; otherwise I can test and merge with the claim button PR

Reply:
earnedamout will never be null since we hard-code it in constants.
 the real fix is to remove earned amount

Reply:
i have tested